common/djangoapps/student/management/commands/recover_account.py

```diff
@@ -84,6 +86,15 @@ def handle(self, *args, **options):
                 user = get_user_model().objects.get(Q(username__iexact=username) | Q(email__iexact=current_email))
                 user.email = desired_email
                 user.save()
+                tracker.emit(
+                    PASSWORD_RESET_INITIATED,
+                    {
+                        "email": user.email,
+                        "user_id": user.id,
+                        "old_email": current_email,
+                        "source": "Account Recovery Management Command",
+                    }
+                )
```

Comment:
Is it necessary to pass user email in event props?

Reply:
Most CRs reference it so from an investigation pov its necessary.

Reply:
I think user id is enough. We don't want to pass unnecessary PII in Segment events. The email doesn't provide any additional insight here in my opinion.

Reply:
@zainab-amir @syedsajjadkazmii I've updated the PR to reflect this, please review. Thanks!